FirstClasses/average/src/average.erl

```diff
@@ -3,4 +3,49 @@
 -export([average/1]).
 
 average(_List) ->
-    ok.
+    case _List of 
+        [] ->
+            0;
+        _ ->
+            sum(_List) / list_lenght(_List)
+    end.
+
+sum(List) ->
```

Comment:
I really like the way you wrapped these functions, it makes comparing implementations very easy :)

Reply:
thank you